File: Easy/Java/2643. Row With Maximum Ones.java
class Solution {
    public int[] rowAndMaximumOnes(int[][] mat) {
        int row = 0;
        int count = 0;

        for(int i = 0; i < mat.length; i++){
            int temp = 0;
            for(int j = 0; j < mat[0].length; j++){
                if(mat[i][j] == 1)
                    temp++;
            }
            if(count < temp){
                row = i;
                count = temp;
            }
        }

        return new int[] {row, count};
    }
}

File: Easy/Python/2643. Row With Maximum Ones.py
class Solution(object):
    def rowAndMaximumOnes(self, mat):
        row = 0
        count = 0

        for i in range(len(mat)):
            temp = 0
            for j in range(len(mat[0])):
                if mat[i][j] == 1:
                    temp += 1
            if count < temp:
                row = i
                count = temp

        return [row, count]
